dev-cpp/tomlplusplus/metadata.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+    <!-- maintainer needed -->
```

Comment:
@MrRoy do you want to help to maintain this package, as it's a dependency of hyprland?

Reply:
@Blackburn29 any reason you didn't put yourself as maintainer?
I don't mind taking it, but given that it's your ebuild, I'd prefer if you took it.

Reply:
My time is limited, but I can try and maintain it if need be

Reply:
> My time is limited, but I can try and maintain it if need be

thanks @Blackburn29, I will just add @MrRoy for now (you can always send a PR later if needed)